main.go

```diff
@@ -49,7 +53,6 @@ func main() {
 	log.SetEnableDebugLog(configs.VerboseLog)
 
 	fmt.Println()
-	successful := true
```

Comment:
The previous test run status calculation, marked a test run as failed if it had a none success status at least once.
This caused an issue with retried tests, even if a rerun had a successful test run the Step reported a failing test run status.